src/controllers/launchpadController.ts

```diff
@@ -0,0 +1,51 @@
+import { RequestHandler } from "express";
+import * as launchpadService from "../services/launchpadService.js";
+
+
+
+/**
+ * Retrieves all launchpads and sends them in the response.
+ * 
+ * @param req - The request object.
+ * @param res - The response object.
+ * @param next - The next middleware function.
+ * 
+ * @returns A JSON response with the list of all launchpads.
+ * 
+ * @throws Passes any errors to the next function.
+ */
+export const getAllLaunchpads: RequestHandler = async (req, res, next) => {
```

Comment:
Сега со typescript може да ставеш types на сите параметри/функции. Пример тука би било

```suggestion
export const getAllLaunchpads: RequestHandler = async (req: Request, res: Response, next: Next) => {
```

Ќе треба точните types да ги најдеш од експрес и да ги импортираш